packages/security_center/lib/disk_encryption/disk_encryption_page.dart

```diff
@@ -17,6 +17,8 @@ const _learnMoreUrl =
     'https://discourse.ubuntu.com/t/hardware-backed-encryption-and-recovery-keys-in-ubuntu-desktop/58243';
 const defaultRecoveryKeyFileName = 'recovery-key.txt';
 
+const actionButtonSize = Size(100, 40);
```

Comment:
Is this minimum button size something we're supposed to implement for all buttons? If so, can this be a global theme override? (We should then also implement this in yaru)

Reply:
I'd also be careful with this, yep. Is there any way we can set the size of the button with spinner dynamically, so it preserves the previous size when it had a text label?